lib/src/database/sembast_database.dart

```diff
@@ -79,6 +79,44 @@ class SembastDatabase extends Database {
     }
   }
 
+  Future<void> copySnapShotAndRefreshTables(List<String> tableNames, String dbAssetPath,
+      List<String> manifest) async {
+    if (dbAssetPath?.isNotEmpty != true ||
+        manifest?.isNotEmpty != true) {
+      return;
+    }
+    var dir;
+    if (!UniversalPlatform.isWeb) {
+      // get document directory
+      final documentPath = await getApplicationSupportDirectory();
+      await documentPath.create(recursive: true);
+      dir = documentPath.path;
+    }
+    final futures = <Future>[];
+    for (final tableName in tableNames) {
```

Comment:
use `Queue` like `Service.dart`
Remember to call `unawaited(_syncQueue.add(() => Future.value()));` before `queue.onComplete`